pkg/relaymonitor/registrations/config.go

```diff
@@ -0,0 +1,37 @@
+package registrations
+
+import (
+	"github.com/ethpandaops/beacon/pkg/human"
+	"github.com/pkg/errors"
+)
+
+type Config struct {
+	// Enabled is the flag to enable the validator registration monitor
+	Enabled *bool `yaml:"enabled" default:"false"`
```

Comment:
Does we need it to a pointer if we default to false? Would avoid dereferencing when using?

Reply:
I can't remember the exact reason off the top of my head, but it's something to do with the yaml defaults library we're using. I think if you have `default: true` it becomes impossible to set it to `false` in yaml without using a pointer. We're defaulting to `false` here, but we've generally just kept them as pointers to save potential headache if we flip it in the future. Good catch tho

Reply:
Ahhhhh yep makes sense. Tyty